setup.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function
-from pkg_resources import parse_version, SetuptoolsLegacyVersion
+from pkg_resources import parse_version
```

Comment:
Replace these two lines with:
```
from pkg_resources import parse_version
try:
    from pkg_resources import SetuptoolsLegacyVersion as _LegacyVersion
except ImportError as e:
    if 'SetuptoolsLegacyVersion' in e.message:
        from packaging.version import LegacyVersion as _LegacyVersion
    else:
        raise  
```

This will not break on devices with low versions of pip (like macbooks)